src/services/UserService.ts

```diff
@@ -50,20 +51,32 @@ export class UserService {
     });
   }
 
-
-  public async savePost(id: Uuid, postId: Uuid): Promise<PostModel> {
+  public async savePost(params: PostAndUserUuidParam): Promise<PostModel> {
     return this.transactions.readOnly(async (transactionalEntityManager) => {
       const postRepository = Repositories.post(transactionalEntityManager);
```

Comment:
I know you didn't change this in this PR but it's weird to change post then user, post then user. Do everything you need with the user first, then do everything you need to do with the post.